README.md

```diff
@@ -51,11 +51,10 @@ For instructions on installing Scope on [Kubernetes](https://www.weave.works/doc
 If you have any questions about, feedback for or problems with Scope:
 
 - Read [the Weave Scope docs](https://www.weave.works/docs/scope/latest/introducing/).
-- Invite yourself to the <a href="https://weaveworks.github.io/community-slack/" target="_blank"> #weave-community </a> slack channel.
-- Ask a question on the <a href="https://weave-community.slack.com/messages/general/"> #weave-community</a> slack channel.
-- Join the <a href="https://www.meetup.com/pro/Weave/"> Weave User Group </a> and get invited to online talks, hands-on training and meetups in your area.
-- Send an email to <a href="mailto:weave-users@weave.works">weave-users@weave.works</a>
-- <a href="https://github.com/weaveworks/scope/issues/new">File an issue.</a>
-
-Your feedback is always welcome!
+- Invite yourself to the <a href="https://weaveworks.github.io/community-slack/" target="_blank">Weave community</a> slack.
```

Comment:
Should we also mention the weekly meetings or wait once we have done a few?

Reply:
Nice one! Pointing to scope community meeting doc.

Reply:
Thanks!